package.json

```diff
@@ -200,6 +200,7 @@
     "karma-jasmine-html-reporter": "2.1.0",
     "ng-packagr": "17.3.0",
     "nx": "18.3.5",
+    "nx-cloud": "18.0.1",
```

Comment:
Just checking... intentional?

Reply:
I was seeing builds being throttled because we weren't on NX 19. I created a separate PR for that #2391.